### DISCUSSION TITLE
I would like to ask whether treeview supports filtering

### DISCUSSION THREAD

**Discussion by dddssw** [microsoft/vscode-discussions#2387] [Extension Development QnA]
![image](https://github.com/user-attachments/assets/61880b34-b510-43a0-bbf2-0e88373fd3fd)
Just like here it can become an input box, and then fuzzy search shows the following tree nodes. Or have other ideas, thank .

**Comment by gjsjohnmurray** [comment]
Ctrl/cmd+Alt+F gets you this.

FYI unless you are developing VS Code extensions this is the wrong forum for you. Please see the pinned READ FIRST thread for guidance.
